app/models/no_cms/blocks/block.rb

```diff
@@ -23,6 +23,16 @@ def fields_configuration
       layout_config.fields
     end
 
+    ##
+    # Get CSS files relative path for the current block
+    def css_files
+      layout_config.css_templates.map do |css_template|
+        media_type = css_template.split("_").last
+        ["#{NoCms::Blocks.css_blocks_folder}/#{self.layout}/#{css_template}", NoCms::Blocks.send("css_mediaquery_#{media_type}")]
```

Comment:
It returns not only the CSS path but also the mediaquery?  We should make it clear in the method comment